tests/test_with_oauth.rs

```diff
@@ -415,7 +416,7 @@ async fn test_recommendations() {
         .recommendations(
             &payload,
             Some(seed_artists),
-            None,
+            None::<Box<dyn Iterator<Item = &str>>>,
```

Comment:
You can use something like `::<&[_]>` to make it simpler

Reply:
If I replace it with `::<&[_]>`, the compiler will complain:

```rust
the size for values of type `str` cannot be known at compilation time
the trait `Sized` is not implemented for `str`
required because of the requirements on the impl of `IntoIterator` for `&[str]`
```

Reply:
Maybe `::<&[&str]>`?

Reply:
fixed

Reply:
If I replace it with `::<&[&str]>`, the compiler complains:

```log
type mismatch resolving `<&[&str] as IntoIterator>::Item == &str` expected reference `&&str`    found reference `&str`
```

Finally, I replaced it with `::Vec<&str>`, it works